api/health/service.go

```diff
@@ -22,6 +23,10 @@ type APIReply struct {
 	Healthy bool              `json:"healthy"`
 }
 
+type HealthArgs struct {
+	SubnetIDs []ids.ID `json:"subnetIDs"`
+}
+
 // Readiness returns if the node has finished initialization
 func (s *Service) Readiness(_ *http.Request, _ *struct{}, reply *APIReply) error {
```

Comment:
Would it make sense for us to support tags in the readiness + liveness checks too?

Reply:
we never actually registered them with a tag, but yeah we can add them since we're here.